app/internal/restore_events.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+from typing import List
+
+from app.database.models import Event, UserEvent
+
+
+def delete_events_after_optionals_num_days(days, session):
+    date_to_delete = datetime.now() - timedelta(days=days)
+
+    user_events_ids_to_be_deleted = (session.query(UserEvent).join(Event).
+                                     filter(
+        Event.deleted_date < date_to_delete).all())
+
+    for id_to_be_deleted in user_events_ids_to_be_deleted:
+        (session.query(UserEvent).filter(UserEvent.id ==
+                                         id_to_be_deleted.id).delete())
+
+    session.query(Event).filter(Event.deleted_date < date_to_delete).delete()
+    session.commit()
+
+
+def get_events_ids_to_restored(events_data: List) -> List[str]:
+    ids = []
+    check_element_name = 'check'
+    check_element_on_value = 'on'
+
+    is_checkbox_element_is_on = False
```

Comment:
You should probably use shorter variable names, for example: instead of is_checkbox_element_is_on - is_checkbox_on.

Reply:
Thanks :)